frontend/src/common/queries/auth.ts

```diff
@@ -37,7 +36,7 @@ async function fetchUserInfo(
 const AUTH_TOKEN_CACHE_TIME_MS = 8 * 60 * 60 * 1000;
 
 export function useUserInfo(hasAuth: boolean) {
-  return useQuery([USE_USER_INFO, { hasAuth }], fetchUserInfo, {
+  return useQuery([USE_USER_INFO, { hasAuth }], () => fetchUserInfo(hasAuth), {
```

Comment:
https://react-query.tanstack.com/guides/migrating-to-react-query-3#query-key-partspieces-are-no-longer-automatically-spread-to-the-query-function